src/sound_win32.c

```diff
@@ -34,7 +34,7 @@ static int num_buffers;
 
 static void show_error(int res)
 {
-	char *msg;
+	const char *msg;
```

Comment:
Yuck, why doesn't the old version emit a warning?

Reply:
`-Wwrite-strings` isn't enabled in C-only mode. (Maybe we should add it.)